External/FEXCore/Source/Utils/AllocatorOverride.cpp

```diff
@@ -0,0 +1,159 @@
+#include <FEXCore/Utils/Allocator.h>
+#include <FEXCore/Utils/CompilerDefs.h>
+#include <FEXCore/Utils/LogManager.h>
+
+#include <fmt/format.h>
+
+#include <cstddef>
+#include <cstdint>
+#include <fstream>
+#include <sstream>
+#include <unistd.h>
+
+extern "C" {
+#ifdef GLIBC_ALLOCATOR_FAULT
```

Comment:
Preprocessor guard not needed since this file is only compiled if this is true. (Could turn the `!defined` case into a hard error maybe)

Reply:
Resolved.